xml/System.Diagnostics.Tracing/IncrementingPollingCounter.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>IncrementingPollingCounter is a variant of EventCounter for variables that are ever-increasing.  Ex) # of exceptions in the runtime. It does not calculate statistics like mean, standard deviation, etc. because it only accumulates the counter value. Unlike IncrementingEventCounter, this takes in a polling callback that it can call to update  its own metric periodically.</summary>
```

Comment:
```suggestion
    <summary><see cref="T:System.Diagnostics.Tracing.IncrementingPollingCounter" /> is a variant of <see cref="T:System.Diagnostics.Tracing.EventCounter" /> for variables that are ever-increasing, like the number of exceptions in the runtime, for example.
    
    It does not calculate statistics like mean, standard deviation, etc. because it only accumulates the counter value. Unlike <see cref="T:System.Diagnostics.Tracing.IncrementingEventCounter" />, this takes in a polling callback that it can call to update  its own metric periodically.</summary>
```